sigsci/manifest.json

```diff
@@ -6,8 +6,8 @@
   "maintainer": "info@signalsciences.com",
   "display_name": "Signal Sciences",
   "short_description": "Collect data from Signal Sciences to see anomalies and block attacks",
-  "metric_prefix": "sigsci.",
```

Comment:
What's the rationale for removing this option? (The value seems correct both here and in `rbltracker`)

Reply:
For `rbltracker`, the integration doesn't submit any metric at all.
For `sigsci` this is an oversight, I assumed the same because of the empty `metadata.csv`

Reply:
Updated